include/fcl/BVH/BVH_model.h

```diff
@@ -268,10 +268,10 @@ class BVHModel : public CollisionGeometry,
   BVHBuildState build_state;
 
   /// @brief Split rule to split one BV node into two children
-  boost::shared_ptr<BVSplitterBase<BV> > bv_splitter;
+  std::shared_ptr<BVSplitterBase<BV> > bv_splitter;
 
   /// @brief Fitting rule to fit a BV node to a set of geometry primitives
-  boost::shared_ptr<BVFitterBase<BV> > bv_fitter;
+  std::shared_ptr<BVFitterBase<BV> > bv_fitter;
```

Comment:
As far as I know, we might don't need to bump the major version number because FCL is still in 0.x.x.